module/handler/fast_forward.py

```diff
@@ -190,27 +185,86 @@ def triggered_map_stop(self):
 
         return False
 
+    def _set_2x_book_status(self, status, check_button, box_button, skip_first_screenshot=True):
+        """
+        Set appropriate 2x book setting
+        with corresponding status and buttons
+        Built with retry mechanism that limits to 3
+        attempts that span 3 second intervals each
+
+        Args:
+            status (string):
+                on or off
+            check_button (Button):
+                button to check before attempting to click
+            box_button (Button):
+                button to click and image color count against
+            skip_first_screenshot (bool):
+                namesake
+
+        Returns:
+            bool:
+                True if detected having set correctly
+                False can occur for 2 reasons either
+                assets insufficient to detect properly
+                or 2x book setting is absent
+
+        """
+        clicked_timeout = Timer(3, count=6)
+        clicked_threshold = 3
+        while 1:
+            if skip_first_screenshot:
+                skip_first_screenshot = False
+            else:
+                self.device.screenshot()
+
+            if clicked_threshold < 0:
+                break
+
+            if clicked_timeout.reached():
+                if self.appear(check_button):
+                    enabled = self.image_color_count(box_button, color=(156, 255, 82), threshold=221, count=20)
+                    if ((status == 'on' and enabled) or (status == 'off' and not enabled)):
+                        return True
+                    if ((status == 'on' and not enabled) or (status == 'off' and enabled)):
+                        self.device.click(box_button)
+
+                clicked_timeout.reset()
+                clicked_threshold -= 1
+
+        logger.warn(f'Wait time has expired; Cannot set 2x book setting')
+        return False

```

Comment:
Use `logger.warning()` instread of `logger.warn()`.
https://stackoverflow.com/questions/15539937/whats-the-difference-between-logging-warn-and-logging-warning-in-python/15655674